packages/app/src/components/Root/LogoFull.tsx

```diff
@@ -16,14 +16,15 @@
 
 import React from 'react';
 import { makeStyles } from '@material-ui/core';
+import teal from '@material-ui/core/colors/teal';
 
 const useStyles = makeStyles({
   svg: {
     width: 'auto',
     height: 30,
   },
   path: {
-    fill: '#7df3e1',
+    fill: teal.A200,
```

Comment:
Lost this comment apparently, should've gone with the last batch 😅 

This is in example of where I'm not a fan of this change, because `#7df3e1` is the color of the Backstage Logo, we shouldn't be changing that to a close approximate

Reply:
Yeah I expected to have to revert back the colors related to branding, and maybe some icons as well

Reply:
Reverted